PRIMER.md

```diff
@@ -363,9 +363,9 @@ Results in:
 
 Polymer supports extending custom element prototypes with shared code modules called "behaviors".
 
-A behavior is simply an object that looks very similar to a typical Polymer prototype.  It may define lifecycle callbacks, `properties`, `hostAttributes`, or other features described later in this document like `observers` and `listeners`.  To add a behavior to a Polymer element definition, include it in a `behaviors` array on the prototype.
+A behavior is simply an object that looks very similar to a typical Polymer prototype.  It may define lifecycle callbacks, `properties`, `hostAttributes`, or other features described later in this document like `observers` and `listeners`.  To add a behavior to a Polymer element definition, include it in a `behaviors` array on the prototype.  If multiple behaviors for the same element define the same function, only the last behavior's function is used.
```

Comment:
This sentence is before the sentence that describes adding arbitrary (non-lifecycle callback) functions.

If it's specifically about these functions, it should move to the next paragraph. If it's talking about the `properties` object, etc., should be rephrased.

(Also, @sjmiles @kevinpschaaf -- I don't know how this works. We should clarify what happens if multiple behaviors include a `properties` object, etc. Are each of these merged into the host element's `properties` object, etc.?